http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/AKSInterpreter.scala

```diff
@@ -510,8 +512,14 @@ class AKSInterpreter[F[_]](config: AKSInterpreterConfig,
         // general configs
         raw"fullnameOverride=$appChartPrefix-${release.asString}",
         raw"instrumentationEnabled=${config.coaAppConfig.instrumentationEnabled}"
-      ).mkString(",")
-    )
+      )
+
+    val updatedLs = sourceWorkspaceId match { // TODO move to buildWdsChartOverrideValues after WDS chart migration
+      case Some(value) => ls ::: List(raw"config.sourceWorkspaceId=${value.value}")
```

Comment:
That makes sense to me, mad at scala for having list concatenation defined as ::: though 😡